reviews/source_control/models.py

```diff
@@ -22,3 +24,50 @@ class PullRequest:
     approved: str
     approved_by_others: bool
     labels: List[Label] = field(default_factory=list)
+
+    def render_approved(self) -> str:
+        """Renders the approved status as a colourised string"""
+        if self.approved == "APPROVED":
+            return "[green]Approved[/]"
+        if self.approved == "CHANGES_REQUESTED":
+            return "[red]Changes Requested[/]"
+        return ""
+
+    def render_approved_by_others(self) -> str:
+        """Renders approved_by_others flag as a colourised string"""
+        return "[green]Ready[/]" if self.approved_by_others else ""
+
+    def render_labels(self, colour_map: Dict[str, str]) -> str:
+        """Renders the labels as a joined colourised string"""
+        return ", ".join(
+            [
+                f"{colour_map.get(label.name.lower(), '[white]')}{label.name}"
+                for label in self.labels
+            ]
+        )
+
+    def render_title(self, org: str, repository: str) -> str:
+        """Renders the title as a colourised string"""
+        colour, title = "", self.title
```

Comment:
NAB: to improve readability it would be nice to split this out into two lines

```python
        colour = ""
        title = self.title
```